indico/legacy/pdfinterface/conference.py

```diff
@@ -139,8 +139,13 @@ def getBody(self, story=None):
         styles = _get_sans_style_sheet()
         style = styles["Normal"]
         style.alignment = TA_JUSTIFY
-        p = Paragraph(escape(track_settings.get(self.event, 'program').encode('utf-8')), style)
-        story.append(p)
+
+        event_program = render_markdown(track_settings.get(self.event, 'program'))
+        for para in event_program.split('\n'):
+            if para.startswith('<p'):
```

Comment:
what about regex matching `<p[ >]` to avoid e.g. `<pre>` from matching here as well?

Reply:
yeah, good point